src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -6,7 +6,7 @@
       color="primary"
       class="btn-outlined-primary"
       :disabled="isAddingAmalgamatingBusiness || isAddingAmalgamatingForeignBusiness"
-      @click="onAddBusinessClick()"
+      @click="isAddingAmalgamatingBusiness = true"
```

Comment:
This seems simpler. I'm trying it out to see if I like it better. (So far, so good.)

Reply:
This works fine. Leaving it.